packages/world/ts/config/v2/worldWithShorthands.ts

```diff
@@ -54,7 +54,7 @@ export type validateNamespacesWithShorthands<namespaces, scope extends Scope = A
   [namespace in keyof namespaces]: {
     [key in keyof namespaces[namespace]]: key extends "tables"
       ? validateTablesWithShorthands<namespaces[namespace][key], scope>
-      : validateNamespaces<namespaces[namespace], scope>[key];
+      : validateNamespace<namespaces[namespace], scope>[key];
```

Comment:
need to remember to add this to `validateWorld` too

Reply:
will do that in [yonadaaa/fix-tables-without-namespace](https://github.com/latticexyz/mud/pull/2880)